packages/noya-designsystem/src/components/ColorPicker.tsx

```diff
@@ -23,14 +24,23 @@ const colorModel: ColorModel<RgbaColor> = {
 
 interface Props {
   value: Sketch.Color;
-  onChange: (color: Sketch.Color) => void;
+  gradients?: Sketch.GradientStop[];
```

Comment:
I think this component would be simpler if we had a separate version specifically for gradients. E.g. `ColorPicker.tsx`, `GradientPicker.tsx`. I think the only component where we should combine everything into one is `ColorInputFieldWithPicker`, for the popup to work correctly

Reply:
Ohh!! Okey I'll try separating them 👍